Gemfile

```diff
@@ -24,3 +24,5 @@ end
 
 # Performance-booster for watching directories on Windows
 gem "wdm", "~> 0.1.1", :install_if => Gem.win_platform?
+
+gem "webrick", "~> 1.7"
```

Comment:
[`jekyll:master`][jekyll] has fixed `webrick` dependency on Ruby 3.0, so we don't need to add any more

[jekyll]: https://github.com/jekyll/jekyll/tree/master